medical_patient_disease/views/medical_patient_disease_view.xml

```diff
@@ -162,7 +162,7 @@
                             </group>
                         </group>
                     </group>
-                    <group name="state" attrs="{'invisible': [('id','=',False)]}">
```

Comment:
I'm getting an error when I try created_at: `Uncaught Error: Unknown field created_at in domain [["created_at","=",false]]`However, my current solution, `[('state','=',False)]`, works. Should I keep state, or is there a better way to use created_at?